granary/tests/test_bluesky.py

```diff
@@ -3513,6 +3516,10 @@ def test_create_with_media(self, mock_get, mock_post):
     expected = copy.deepcopy(POST_BSKY_IMAGES)
     del expected['fooOriginalText']
     del expected['fooOriginalUrl']
+    expected['embed']['images'][0]['aspectRatio'] = {
+      'width': 500,
```

Comment:
minor nit, mind using a non-square test image, so that the test can check that width and height end up in the right place?

Reply:
Good spot

Reply:
Done!